cpp/tests/CMakeLists.txt

```diff
@@ -172,16 +173,6 @@ ConfigureTest(INTEROP_TEST
     interop/from_arrow_test.cpp
     interop/dlpack_test.cpp)
 
-###################################################################################################
-# - jit cache tests -------------------------------------------------------------------------------
-ConfigureTest(JITCACHE_TEST
```

Comment:
No longer testing jit cache?

Reply:
jitify2 seems to have its own LRU file cache. I'm not sure how it's activated though.

Reply:
by providing a cache dir to ProgramCache<> constructor. Working on adding that.